datum/docker-compose.yml

```
@@ -0,0 +1,11 @@
+version: "3.7"
+
+services:
+  datum:
+    image: ghcr.io/retropex/datum:v1.3
+    volumes:
+      - type: bind
+        source: ${APP_DATA_DIR}/settings/datum_gateway_config.json
+        target: /app/datum_gateway_config.json
```

Since we're moving `settings` one level deeper inside the `data` directory we can change to:
```suggestion
    volumes:
        - ${APP_DATA_DIR}/data/settings/datum_gateway_config.json:/app/datum_gateway_config.json
```

Do you know if the config file is the only settings file that needs to be persisted? Or is there any other data we need to persist?

---

Only the config file the rest of data is managed by the pool and the node.